EDU-12592: Section override use cases (#1264)
vtexdocs/dev-portal-content: docs/faststore/docs/customization/building-sections/override-use-cases/overview.mdx

---
title: 'Section override use cases'
---

The Section override use cases provide an introduction, instructions for implementation, and results in the storefront, showing how we can implement [section override](https://developers.vtex.com/docs/guides/faststore/overrides-overview). These examples can help you understand and leverage this feature.

| Use case | Description |
| -------- | ----------- |
| [Adding more icons options to the Alert component](https://developers.vtex.com/docs/guides/faststore/override-use-cases-adding-more-icon-options-to-the-alert-component) | Add more icons options to the Alert, allowing editors to choose from a wider selection of icons supported by the `@faststore/ui` [iconography](https://developers.vtex.com/docs/guides/faststore/reference-icons#usage) library.|
| [Adding an image to your Alert component](https://developers.vtex.com/docs/guides/faststore/override-use-cases-adding-an-image-to-the-alert-component) | Make alert messages more engaging by allowing images alongside text content. |

> > ℹ️ In each use case guide, you can get more code details by accessing the [playground.store](https://github.com/vtex-sites/playground.store) repository, which provides more in-depth details of the examples.

Reviewer: ```suggestion
> ℹ️ You can find the code for each use case in the [playground.store](https://github.com/vtex-sites/playground.store) repository, which provides more in-depth details of the examples.
```
